client/gutenberg/editor/browser-url/index.js

```diff
@@ -51,7 +51,7 @@ export class BrowserURL extends Component {
 			prevProps.postStatus === 'auto-draft' &&
 			! endsWith( currentRoute, `/${ postId }` )
 		) {
-			this.props.replaceHistory( `${ currentRoute }/${ postId }` );
+			this.props.replaceHistory( `${ currentRoute }/${ postId }`, true );
```

Comment:
how about we add an inline comment here to indicate what `true` is for and what it sets…

(I tried looking it up but I didn't see it so maybe someone can help here)

Reply:
👍 If we get another prop we can also refactor to an options object.